README.md

```diff
@@ -96,6 +96,11 @@ Notable data types are:
   question mark. This allows you to represent dates such as "some time in
   1973" or "26th of July, but I don't know which year", which can be common
   when entering data without full knowledge of the truth.
+- `inferred`: This is a special type for which you can't create any actual
```

Comment:
suggestion: This is a special, read-only claim type that users cannot create directly. Instead, you define inference rules that automatically display derived claims on a claim’s detail page. For example, given a `child of` claim, a rule can infer a `sibling` claim, based on the fact that siblings always have a shared parent.